backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/RabbitMQHealthCheck.java

```diff
@@ -53,12 +49,14 @@ public ComponentName componentName() {
 
     @Override
     public Result check() {
-        try (Connection connection = connectionFactory.newConnection()) {
-            if (connection.isOpen()) {
-                return Result.healthy(COMPONENT_NAME);
-            }
-            LOGGER.error("The created connection was not opened");
-            return Result.unhealthy(COMPONENT_NAME);
+        try {
+            return rabbitChannelPool.execute(channel -> {
```

Comment:
Wouldn't it make sens to check all channels, then decide the status? Which could be intermediate...

Reply:
there's no such "all channels", you will in any case test a snapshoted list of them, maybe outdated ?
Being able to check liveness of a arbitrary channel means the next channel client will probably be able to use it.